invisible_cities/cities/components.py

```diff
@@ -451,6 +453,16 @@ def mcsensors_from_file(paths     : List[str],
                                                        return_raw = False     ,
                                                        db_file    = db_file   ,
                                                        run_no     = run_number)
+        sns_pos_tbl = 'MC/sns_positions'
+        if is_oldformat_file(file_name): sns_pos_tbl = 'MC/sensor_positions'
+
+        nexus_sns_pos       = pd.read_hdf(file_name, sns_pos_tbl)
+        nexus_pmt_ids       = nexus_sns_pos[nexus_sns_pos.sensor_id < 100].sensor_id.values
+        nexus_pmt_arg_in_db = np.argwhere([sid in pmt_ids for sid in nexus_pmt_ids])
+
+        if len(nexus_pmt_ids) != len(nexus_pmt_arg_in_db):
```

Comment:
A more concise way to express that every number in a sequence is present in another sequence, using a `pd.Series`, is:
```
nexus_pmt_ids = nexus_sns_pos[nexus_sns_pos.sensor_id < 100].sensor_id
if nexus_pmt_ids.isin(pmt_ids).all():
```